modules/gatsby/src/components/layout/table-of-contents.jsx

```diff
@@ -81,87 +84,106 @@ function getHeight(route) {
 // effectively change routes. If no path is passed or if the path is not
 // usable then it just renders a div. That should not be the case
 
-const SafeLink = ({ className, name, path }) => {
-  if (!path || typeof path !== 'string') {
-    return (
-      <div className={className} title={name}>
-        {name}
-      </div>
-    );
+const Img = ({ collapsed, expanded, style }) => {
```

Comment:
Maybe a one line comment above each of these helper components to state the intended function?